src/index.rs

```diff
@@ -1133,18 +1133,19 @@ mod tests {
         assert!(new2 > new1);
     }
 
+    const CREATE: i32 = 1;
+    const UPDATE: i32 = 2;
+    const DELETE: i32 = 3;
+    const MOVE: i32 = 5;
+
     const FILE_DIR_1: &str = "folder_1";
     const FILE_DIR_2: &str = "folder_2";
     const FILE_NAME: &str = "test_";
-    const FILE_COUNT: i32 = 10;
+    const FILE_COUNT: i32 = 3;
 
-    fn generate_random_update(
-        root_path: PathBuf,
-        folder_name: Option<&str>,
-        name: Option<&str>,
-    ) -> i32 {
+    fn generate_random_update(root_path: PathBuf, name: Option<&str>) -> i32 {
         let mut rng = rand::thread_rng();
-        let mut rnd_num = rng.gen_range(1..=4);
+        let rand_num: i32 = rng.gen_range(0..10);
```

Comment:
What is `10`? Better to declare the constant.

Reply:
> What is `10`? Better to declare the constant.

I update this code.
let rand_num: i32 = rng.gen_range(0..ACTIONCNT);